src/session/secret_chat.rs

```diff
@@ -72,7 +72,9 @@ mod imp {
                         "The state of this secret chat",
                         SecretChatState::static_type(),
                         SecretChatState::default() as i32,
-                        glib::ParamFlags::READWRITE | glib::ParamFlags::EXPLICIT_NOTIFY,
+                        glib::ParamFlags::READWRITE
+                            | glib::ParamFlags::CONSTRUCT
```

Comment:
I just noticed that we don't set `last-message` during construction. Maybe we should also consider doing this?